Fabric.Authorization.UnitTests/EDWAdminSyncService/EDWAdminRoleSyncServiceTests.cs

```diff
@@ -335,34 +355,34 @@ public async Task SyncPermissions_NullUserArrayDoesNotThrowsExceptionAsync()
                 new List<User>() {
                     new User("testSubjectId8", "windows")
                     {
-                        Roles = new Role[] { new Role { Name = "notadmin" } },
-                        Groups = new List<Group>() { DeletedGroup }
+                        Roles = new Role[] { NoAdminRole },
+                        Groups = new List<Group>() { DosAdminsDeletedGroup }
                     },
                     new User("testSubjectId9", "windows")
                     {
-                        Roles = new Role[] { new Role { Name = "notadmin" } },
-                        Groups = new List<Group>() { DeletedGroup }
+                        Roles = new Role[] { NoAdminRole },
+                        Groups = new List<Group>() { DosAdminsDeletedGroup }
                     }
                 },
                 0,
                 2
             },
-            new object[] // If a group is deleted, then remove all non admins
+            new object[] // If DosAdmins group is deleted, then remove all users (including admins)

```

Comment:
Just to follow the same pattern that the previous test was trying to convey, i would add a group/role that is removed and also one that is added, that way it shows that 1 and 1 scenario.